### PR TITLE
Remove the obsolete cmake options

### PR DESCRIPTION
**Pull request by Shixiaowei02**
Currently, the Anakin subgraph has been developed, and the original online cmake compile commands for github anakin needs to be removed.

### CODE REVIEW

File: paddle/fluid/inference/tests/api/CMakeLists.txt [hunk 1]
```diff
@@ -254,27 +254,6 @@ set(BERT_INSTALL_DIR "${INFERENCE_DEMO_INSTALL_DIR}/bert_emb128")
 download_model_and_data(${BERT_INSTALL_DIR} "bert_emb128_model.tar.gz" "bert_data_len20.txt.tar.gz")
 inference_analysis_api_test(test_analyzer_bert ${BERT_INSTALL_DIR} analyzer_bert_tester.cc)
```
Comment (luotao1) [inline paddle/fluid/inference/tests/api/CMakeLists.txt:255]: Do `anakin_rnn1_tester.cc` and `anakin_mobilenet_tester.cc` be removed at the same time?

Comment (Shixiaowei02) [reply paddle/fluid/inference/tests/api/CMakeLists.txt:255]: ok, I removed them.

**Review by luotao1 — APPROVED** [review]
LGTM

**Comment by luotao1** [comment]
Could you remove `WITH_ANAKIN` option in `paddle_build.sh`? @Shixiaowei02

**Pull request merged**